src/topkg_conf.ml

```diff
@@ -370,6 +370,25 @@ let tool ?conf name os = match Topkg_os.Env.var (os_tool_env name os) with
             | Some cmd -> cmd
             | None -> Topkg_cmd.v name
 
+let jobs =
+  let doc = "Allow to run $(docv) commands at once when building." in
+  let absent () = Ok None in
+  discovered_key "jobs" (some int) ~absent ~doc ~docv:"JOBS"
+
+let default_jobs = 4
+let jobs c = match value c jobs with
+| Some n -> n
+| None ->
+    match build_context c with
+    | `Dev ->
+        let getconf = tool "getconf" `Build_os in
```

Comment:
Pushed an update. Tested on `OpenBSD 6.1`, Linux, OS/X via Travis and Win32 via Appveyor.
Added the CI scripts I used too don't know if you want them.